### PR TITLE
Pt 178728252

### PR DESCRIPTION
**Pull request by sri49**

### CODE REVIEW

File: features/insured/individual_signup.feature [hunk 1]
```diff
@@ -3,52 +3,21 @@ Feature: Insured Plan Shopping on Individual market
   Background:
     Given Individual has not signed up as an HBX user
     Given the FAA feature configuration is enabled
-    When Individual visits the Insured portal during open enrollment
+    And Individual market is under open_enrollment period
     Then Individual creates a new HBX account
     Then Individual should see a successful sign up message
```
Comment (bill-transue) [inline features/insured/individual_signup.feature:8]: Why are we still calling these steps? It seems like we're correctly creating the `Patrick Doe` user below, on line 11.

Comment (sri49) [reply features/insured/individual_signup.feature]: Yeah, we don't need that, removed in this commit- https://github.com/ideacrew/enroll/pull/545/commits/23af69a48ba11668c8aca22f8ee9e4abf9a92e0d

File: features/financial_assistance/fa_not_applying.feature [hunk 2]
```diff
@@ -12,7 +12,7 @@ Feature: User is not applying for financial assistance
 
   Scenario: Consumer is not applying for financial assistance
     Given the user navigates to the "Household Info" page with "no" selected
-    When the user clicks on add member button
+    When the user click on add member button
```
Comment (bill-transue) [inline features/financial_assistance/fa_not_applying.feature:15]: Let's not change the step to be less grammatically correct.

**Review by bill-transue — CHANGES_REQUESTED** [review]
In `fa_no_applying.feature` we have a lot of Givens that are being done through the browser. Let's get those done in the background so the step goes faster just like you did in `individual_signup.feature`, which looks great! I also would like know how long `add_sep_read_only.feature` takes to run and how much time it will add to our overall test suite.

**Pull request closed without merging**